### PR TITLE
Prepare for v4.3.1

### PR DESCRIPTION
**Pull request by sebrandon1**
https://github.com/test-network-function/cnf-certification-test-partner/releases/tag/v4.3.1

### CODE REVIEW

**Review by rdavid — APPROVED** [review]

**Review by edcdavid — APPROVED** [review]
Shouldn't we also increment the version.json?

File: docs/runtime-env.md [hunk 1]
```diff
@@ -68,4 +68,4 @@ export TNF_PARTNER_REPO=registry.dfwt5g.lab:5000/testnetworkfunction
 ```
 
 Note that you can also specify the debug pod image to use with `SUPPORT_IMAGE`
-environment variable, default to `debug-partner:4.3.0`.
+environment variable, default to `debug-partner:4.3.1`.
```
Comment (edcdavid) [inline docs/runtime-env.md:71]: Shouldn't we also increment the version.json?

**Comment by sebrandon1** [comment]
Good call I missed that one.  Fixed.

File: docs/runtime-env.md (hunk 1 above)
Comment (sebrandon1) [reply docs/runtime-env.md:71]: Fixed.

**Pull request merged**